src/overlays/actors/ovl_Obj_Warpstone/z_obj_warpstone.c

```diff
@@ -179,11 +179,11 @@ void ObjWarpstone_Draw(Actor* thisx, PlayState* play2) {
         gDPPipeSync(POLY_XLU_DISP++);
         gDPSetPrimColor(POLY_XLU_DISP++, 128, 128, 255, 255, 200, this->dyna.actor.home.rot.x);
         gDPSetEnvColor(POLY_XLU_DISP++, 100, 200, 0, 255);
-        Matrix_RotateZF((((play->gameplayFrames * 1500) & 0xFFFF) * M_PI) / 0x8000, MTXMODE_APPLY);
+        Matrix_RotateZF(BINANG_TO_RAD_ALT2((play->gameplayFrames * 1500) & 0xFFFF), MTXMODE_APPLY);
```

Comment:
Do u16 casts work?

Reply:
It works for the first without the `~`, but breaks on the second. So I suppose the consistency is better. Notably, this grouping of functions and calcs is identical to a block of code in OoT's Farore's Wind